### PR TITLE
fix(color-edit): fix of displaying opacity count below 58%

### PR DESCRIPTION
**Pull request by alexishere24**
- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?

Opacity count input doesnt work correctly with  opacity below 58%

Closes # 

## What is the new behavior?

Opacity count input work correctly

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by splincode** [comment]
why `package-lock.json` was update?

File: projects/addon-editor/components/color-selector/color-edit/color-edit.component.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ export class TuiColorEditComponent {
     }
 
     get opacity(): number {
-        return this.color[3] * 100;
+        return round(this.color[3] * 100);
```
Comment (waterplea) [inline projects/addon-editor/components/color-selector/color-edit/color-edit.component.ts:44]: Let's just use `Math.round`

**Review by waterplea — CHANGES_REQUESTED** [review]
Small comment and also please rollback the `package-lock` change. Use `npm ci` in future so it does not change the lock file.

**Comment by vladimirpotekhin** [comment]
https://github.com/Tinkoff/taiga-ui/pull/1312

**Pull request closed without merging**